src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(new StringBuilder()

```

Comment:
Using StringBuilder is unnecessary for this simple concatenation. It could be replaced with a simple string concatenation for better readability.

Reply:
Na którejś z lekcji mówiono, że StringBuilder jest lepszy ze względu na to, że nie tworzy zbędnych obiektów. Stąd wybór tego sposobu na stworzenie wiadomości. Już poprawiam.

Reply:
Jeśli w kodzie napiszesz:
`String a = "foo" + "bar";`
To kompilator i tak to zamieni na:
`String a = new StringBuilder("foo").append("bar").toString();`

Klasy `StringBuilder` powinno się używać tam, gdzie konkatenacja `String`ów jest wewnątrz pętli, bo wtedy kompilator wewnątrz pętli zmieni konkatenację na obiekt `StringBuilder` i przy wywołaniu pętli np 10 000 razy, stworzy on co najmniej 10 000 obiektów, co zajmie bardzo dużo pamięci i czasu.

Taka dygresja. :)

Reply:
Dzięki! :) już wszystko jasne.